code/drv_db/src/wattrex_driver_db/drv_db_engine.py

```diff
@@ -41,38 +38,45 @@ class DrvDbSqlEngineC:
 
     __MAX_RESETS = 2
 
-    def __init__(self, config_file='../config/.cred.yaml'):
+    def __init__(self, db_type : DrvDbTypeE, config_file):
         '''
-        Create an connector to the MySQL database server
+        Create an connector to the MySQL or SQLite database server
```

Comment:
It not depends on technology. Distinguishes between master and cache databases.

Reply:
Done